components/Common/PrimaryButton/PrimaryButton.tsx

```diff
@@ -0,0 +1,34 @@
+interface PrimaryButtonProps {
+  text: string;
+  onClick?(): void;
+  buttonLocated?: "lg" | "md" | "sm" | "gray";
```

Comment:
@oscarcorellan let's change this prop name 

```suggestion
  size?: "lg" | "md" | "sm" | "gray";
```